pkg/prometheus/operator.go

```diff
@@ -472,21 +497,14 @@ func (c *Operator) syncNodeEndpoints() error {
 		return errors.Wrap(err, "listing nodes failed")
 	}
 
-	for _, n := range nodes.Items {
-		address, _, err := nodeAddress(n)
-		if err != nil {
-			return errors.Wrapf(err, "failed to determine hostname for node (%s)", n.Name)
+	addresses, errs := getNodeEndpoints(nodes)
```

Comment:
small nit: it's a little funny that the return value of `getNodeEndpoints` is named `addresses`. I think either the func or the value should be renamed so they agree

Reply:
Good point, I'll fix